typing/btype.mli

```diff
@@ -127,12 +127,25 @@ val copy_row:
     bool -> row_desc -> bool -> type_expr -> row_desc
 val copy_kind: field_kind -> field_kind
 
-val save_desc: type_expr -> type_desc -> unit
+module For_copy : sig
+
+  type copy_scope
+        (* A token used to ensure that calls to [save_desc] or [dup_kind]
+           only happen from inside a call to [with_scope].
```

Comment:
This part of the comment is not really correct anymore now that `copy_scope` carries real data. I would say: 

> The private state that the copy primitives below are mutating, which should remain scoped within a single `with_scope` call.

Reply:
Done.